src/index.ts

```diff
@@ -175,7 +177,23 @@ extendEnvironment((hre) => {
             return wallet;
         },
 
-        devnet: lazyObject(() => new DevnetUtils(hre))
+        devnet: lazyObject(() => new DevnetUtils(hre)),
+
+        deployAccountFromABI: async (accountContract, accountType) => {
```

Comment:
Will not be needed if we decide to expose OpenZeppelinAccount class with its constructor.